fhir-server/src/main/java/au/csiro/pathling/aggregate/DrillDownBuilder.java

```diff
@@ -50,10 +50,10 @@ public DrillDownBuilder(@Nonnull final List<Optional<Type>> labels,
   /**
    * Generates the FHIRPath string.
    *
-   * @return A FHIRPath expression
+   * @return A FHIRPath expression, unless there are no groupings or filters
```

Comment:
I guess this is the matter of convention but  rather than no drill down expression it could be also an empty one which would result in all resources being returned when queried.

Reply:
Resolved that we will only supply a drill down expression where one is needed. We will also enforce that empty expressions cannot be supplied to operations - this change will be made as part of #201.